### PR TITLE
STYLING

### PR DESCRIPTION
**Pull request by karanJ2212**
-added styling to the project

### CODE REVIEW

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (HAMMAS-SALEEM) [inline .hintrc:1]: Capture1
- [ ] Your link to your license file is not working becuase you didn't create any link :fearful: Kindly add a license file using these [guildlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush: :+1:

- [ ] After making that license file kindly link that license file in your readme file :thinking:

Comment (HAMMAS-SALEEM) [inline .hintrc:1]: - [ ] You don't have a link to your issues page of your repository. Kindly add a link to your issues page in your readme file to make it descriptive 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @karanJ2212 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .hintrc (hunk 1 above)
Comment (karanJ2212) [reply .hintrc:1]: thanks, done

Comment (karanJ2212) [reply .hintrc:1]: Thanks done

Comment (HAMMAS-SALEEM) [reply .hintrc:1]: It's great on working on the changes 👍

**Review by levy002 — APPROVED** [review]
## Hello @karanJ2212 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Excellent UI :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**